src/components/App.js

```diff
@@ -17,6 +19,8 @@ function App() {
         <Route path="/calendar" component={Calendar} />
         <Route path="/new-event" component={Form} />
         <Route path="/contacts" component={Contacts} />
+        <Route path="/event-details" component={Event} />
```

Comment:
if we want we can probably adjust this to be a match parameter but this is also probably fine for now!